public/app/features/dashboard/components/PanelEditor/OptionsGroup.tsx

```diff
@@ -162,7 +162,7 @@ const getStyles = stylesFactory((theme: GrafanaTheme, isExpanded: boolean, isNes
     ),
     body: cx(
       css`
-        padding: 0 ${theme.spacing.md} ${theme.spacing.md} ${theme.spacing.xl};
+        padding: ${theme.spacing.sm} ${theme.spacing.md} ${theme.spacing.xl} ${theme.spacing.sm};
```

Comment:
```suggestion
        padding: ${theme.spacing.sm} ${theme.spacing.md} ${theme.spacing.xl} ${theme.spacing.md};
```

Shouldn't the last one be `${theme.spacing.md}` if we specify it? Or removed since it will be the same as the second value if we remove it.

Reply:
ah, thanks for catching this, I fixed it first in browser and made a mistake when implementing it code!